doc/api/buffer.md

```diff
@@ -379,9 +379,10 @@ A `TypeError` will be thrown if `size` is not a number.
 
 The `Buffer` module pre-allocates an internal `Buffer` instance of
 size [`Buffer.poolSize`][] that is used as a pool for the fast allocation of new
-`Buffer` instances created using [`Buffer.allocUnsafe()`][] and the deprecated
-`new Buffer(size)` constructor only when `size` is less than or equal to
-`Buffer.poolSize >> 1` (floor of [`Buffer.poolSize`][] divided by two).
+`Buffer` instances created using [`Buffer.allocUnsafe()`][],
+[`Buffer.from(array)`][] and the deprecated `new Buffer(size)` constructor only
```

Comment:
Our style guide encourages serial commas:

```suggestion
[`Buffer.from(array)`][], and the deprecated `new Buffer(size)` constructor only
```

Reply:
@Trott, thanks. Fixed it.